app-catalog/src/components/charts/List.tsx

```diff
@@ -140,14 +145,47 @@ export function ChartsList({ fetchCharts = fetchChartsFromArtifact }) {
     );
   }
 
+  function SettingsLink() {
```

Comment:
SettingsLink could be added in a separate commit in a SettingsLink.tsx file. With a SettingsLink.stories.tsx

Reply:
done!